setup-without-nix.sh

```diff
@@ -41,6 +41,16 @@ set -o pipefail
     ssh-keygen -t ed25519 -f /Volumes/CONFIG/buildkite-agent/sshkey -N ""
   fi
 
+  # install xcode (for git) ugh
+  # inspired by https://gist.github.com/mokagio/b974620ee8dcf5c0671f
+  # and yes, this file is required, or else the command line tools don't show up in softwareupdate -l
+  touch /tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress
+  softwareupdate -i "$(softwareupdate -l | grep "\*.*Command Line" | tail -n 1 | sed 's/^[^C]* //')"
+  if ! git --help &>/dev/null; then
+    # Didn't find command line tools first time, try again?
+    softwareupdate -i "$(softwareupdate -l | grep "\*.*Command Line" | tail -n 1 | sed 's/^[^C]* //')"
```

Comment:
Mmmm does this... help?

Reply:
Yes. One time, I witnessed a machine that booted with this commit, but still did not have `git` installed. So we try again at least once.